modules/python/src2/pycompat.hpp

```diff
@@ -229,7 +229,12 @@ PyObject* pyopencv_from(const TYPE& src)
             ERROR_HANDLER; \
         } \
         CVPY_TYPE_INCREF(pyopencv_##NAME##_TypePtr); \
-        PyModule_AddObject(m, #WNAME, (PyObject *)pyopencv_##NAME##_TypePtr); \
+        if (PyModule_AddObject(m, #WNAME, (PyObject *)pyopencv_##NAME##_TypePtr) < 0) \
+        { \
+            printf("Failed to register a new type: " #WNAME  ", base (" #BASE ")\n"); \
```

Comment:
Should we expect some of these messages on CI? (are there some failures?)

Reply:
No failures on CI is expected.
If this check is failed than there is definitely something wrong with whole bindings code generation.
Reference implementation of `PyModule_AddObject` from CPython repository: [link](https://github.com/python/cpython/blob/f4c03484da59049eb62a9bf7777b963e2267d187/Python/modsupport.c#L670)
All error checking happens in [PyModule_AddObjectRef](https://github.com/python/cpython/blob/f4c03484da59049eb62a9bf7777b963e2267d187/Python/modsupport.c#L637)